lib/Table/TableStyle.ts

```diff
@@ -297,13 +324,7 @@ export default class TableStyle {
   get numberOfBins(): number {
     const colorColumn = this.colorColumn;
     if (colorColumn === undefined) return this.binMaximums.length;
-    if (
-      colorColumn.type === TableColumnType.enum ||
-      colorColumn.type === TableColumnType.region ||
-      colorColumn.type === TableColumnType.text
-    ) {
-      return colorColumn.uniqueValues.values.length;
-    } else if (colorColumn.type === TableColumnType.scalar) {
+    if (colorColumn.type === TableColumnType.scalar) {
```

Comment:
This was problematic - it was creating many many colour bins for some columns - which was useless